deploy/dmaas-operator.yaml

```diff
@@ -0,0 +1,1212 @@
+              type: string
+            updatedAnnotatedList:
+              description: UpdatedAnnotatedList represents the list of resources annotated
+                successfully
+              items:
+                description: TargetResource represents the k8s resource information
+                  used for pre-backup action
+                properties:
+                  apiversion:
+                    description: APIVersion of the resource
+                    type: string
+                  kind:
+                    description: Kind of the resource
+                    type: string
+                  name:
+                    description: Name of the resource
+                    items:
+                      type: string
+                    type: array
+                  namespace:
+                    description: Namespace of the resource
+                    type: string
+                required:
+                - kind
+                type: object
+              nullable: true
+              type: array
+            updatedSelectedList:
+              description: UpdatedSelectedList represents the list of resources labeled
+                successfully
+              items:
+                description: TargetResource represents the k8s resource information
+                  used for pre-backup action
+                properties:
+                  apiversion:
+                    description: APIVersion of the resource
+                    type: string
+                  kind:
+                    description: Kind of the resource
+                    type: string
+                  name:
+                    description: Name of the resource
+                    items:
+                      type: string
+                    type: array
+                  namespace:
+                    description: Namespace of the resource
+                    type: string
+                required:
+                - kind
+                type: object
+              nullable: true
+              type: array
+          required:
+          - phase
+          type: object
+      type: object
+  version: v1alpha1
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: dmaas
+  labels:
+    component: dmaas
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: dmaas-operator
+  labels:
+    name: dmaas-operator
+    component: dmaas
+  namespace: dmaas
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: dmaas-operator
+  labels:
+    name: dmaas-operator
+    component: dmaas
+  namespace: dmaas
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: dmaas-operator
+    namespace: dmaas
+---
+kind: Deployment
```

Comment:
similar deployment to this is in dmaas-resources.yaml also.. same is the case with other objects ClusterRoleBinding, ServiceAccount, Namespace.. why so?

Reply:
ok.. as per update-codegen.sh script, dmaas-resource.yaml is appended to dmaas-operator.yaml to get all of them at a single place